packages/remote-config/src/abt/experiment.ts

```diff
@@ -45,32 +58,48 @@ export class Experiment {
     currentActiveExperiments: Set<string>,
     experimentInfoMap: Map<string, FirebaseExperimentDescription>
   ): void {
+    const customProperty: Record<string, string | null> = {};
     for (const [experimentId, experimentInfo] of experimentInfoMap.entries()) {
       if (!currentActiveExperiments.has(experimentId)) {
-        this.addExperimentToAnalytics(experimentId, experimentInfo.variantId);
+        customProperty[experimentId] = experimentInfo.variantId;
       }
     }
+    this.addExperimentToAnalytics(customProperty);
   }
 
   private removeInactiveExperiments(
     currentActiveExperiments: Set<string>,
     experimentInfoMap: Map<string, FirebaseExperimentDescription>
   ): void {
+    const customProperty: Record<string, string | null> = {};
     for (const experimentId of currentActiveExperiments) {
       if (!experimentInfoMap.has(experimentId)) {
-        this.removeExperimentFromAnalytics(experimentId);
+        customProperty[experimentId] = null;
       }
     }
+    this.addExperimentToAnalytics(customProperty);
```

Comment:
Sorry if this is a duplicate comment - I can't see the previous one I left anymore - but this should be `removeExperimentFromAnalytics`, right?

Reply:
This was removed as per https://github.com/firebase/firebase-js-sdk/pull/9278#discussion_r2382448445